Assets/Vimeo/Scripts/Services/VimeoUploader.cs

```diff
@@ -161,5 +190,13 @@ public void UploadNextChunk()
                 }
             }
         }
+
+        private bool HasChunksLeftToUpload()
+        {
+            if (currentChunkIndex < m_chunks.Count) {
```

Comment:
a more future proof way of checking would be to create a method called `GetTotalRemainingChunks` which loops through to check status. e.g.

```
private bool HasPendingChunksToUpload()
{
   return GetTotalRemainingChunks() == 0;
}
```